### PR TITLE
Round 2 of integration test fixes

### PR DESCRIPTION
**Pull request by haircommander**
Clean up the integration tests by cleaning /tmp more consistently (more details in commit message) and remove the `|| true` hack to work on getting to the bottom of those flakes.

### CODE REVIEW

File: test/ctr_seccomp.bats [hunk 1]
```diff
@@ -37,9 +41,9 @@ function teardown() {
 	echo "$output"
 	[ "$status" -eq 0 ]
 
-	cleanup_ctrs
-	cleanup_pods
-	stop_crio
+	#cleanup_ctrs
```
Comment (haircommander) [inline test/ctr_seccomp.bats]: TODO: there seemed to be extra flakes on tests where these functions were called twice. I haven't looked deep into it yet, and am not sure I was correct.

**Comment by haircommander** [comment]
SO these fixes haven't actually fixed the problem, but I found a bunch of places that needed cleaning up anyway. I will continue on this quest, but once e2e is fixed I'd like these merged. I put the || true back to return to stability

**Comment by haircommander** [comment]
while this solution is imcomplete, it does address some issues i found and I'd like to get it in.
@mrunalp @umohnani8 @saschagrunert @rhatdan PTAL

File: test/helpers.bash [hunk 2]
```diff
@@ -434,8 +439,7 @@ function cleanup_test() {
 	cleanup_pods
 	stop_crio
 	cleanup_lvm
-	# Best effort remove the test dir, as sometime it is still busy
-	rm -rf "$TESTDIR" || true
+	rm -r "$TESTDIR" || true
```
Comment (saschagrunert) [inline test/helpers.bash]: Is this still an issue, so can we remove the `|| true` now?

Comment (haircommander) [reply test/helpers.bash]: unfortunately it is still an issue

Comment (saschagrunert) [reply test/helpers.bash]: Okay, I will link it in https://github.com/cri-o/cri-o/issues/2665 Let's get these cleanups in nevertheless :)

**Comment by umohnani8** [comment]
LGTM

File: test/status.bats [hunk 3]
```diff
@@ -3,6 +3,7 @@
 load helpers
 
 function setup() {
+	setup_test
```
Comment (mrunalp) [inline test/status.bats]: nit: spacing but can fix in next update.

**Comment by haircommander** [comment]
those are new and exciting failures that I am hoping are flakes
/retest

**Comment by saschagrunert** [comment]
Unfortunately this now needs a rebase.

**Comment by haircommander** [comment]
rebased and unified the whitespace in test/helpers as an added bonus :)

**Comment by haircommander** [comment]
rebased again, will need another lgtm @rhatdan :)

**Pull request merged**